pv_site_api/main.py

```diff
@@ -386,6 +387,8 @@ def get_pv_estimate_clearsky_many_sites(
         tilt = site.tilt or 0
         orientation = site.orientation or 180
 
+        module_capacity = site.module_capacity_kw or site.inverter_capacity_kw * 1.5
```

Comment:
Should there be a comment added for this scaling on inverter capacity?

Reply:
Yep, added